### PR TITLE
Fix libxml2 CVEs

### PR DESCRIPTION
**Pull request by MateuszMalisz**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

I've updated libxml2 to version 2.9.12-1 to fix various CVEs.

###### Change Log  

- Update to version 2.9.12 
- Fixes CVE-2021-3517 
- Fixes CVE-2021-3518
- Fixes CVE-2021-3537

###### Does this affect the toolchain?  

**YES**

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2021-3517 
- https://nvd.nist.gov/vuln/detail/CVE-2021-3518 
- https://nvd.nist.gov/vuln/detail/CVE-2021-3537 

###### Test Methodology

- Local build

### CODE REVIEW

**Comment by anphel31** [comment]
You'll also need to update:
/toolkit/resources/manifests/package/pkggen_core_x86_64.txt
/toolkit/resources/manifests/package/pkggen_core_aarch64.txt
/toolkit/resources/manifests/package/toolchain_aarch64.txt
/toolkit/resources/manifests/package/toolchain_x86_64.txt
/cgmanifest.json

File: SPECS/libxml2/libxml2.spec [hunk 1]
```diff
@@ -105,6 +102,9 @@ rm -rf %{buildroot}/*
 %{_libdir}/cmake/libxml2/libxml2-config.cmake
 
 %changelog
+* Thu May 27 2021 Mateusz Malisz <mamalisz@microsoft.com> - 2.9.12-1
+- Update to version 2.9.12
```
Comment (jslobodzian) [inline SPECS/libxml2/libxml2.spec:106]: Please add the reason why you are updating.  To fix a CVE.

**Comment by jslobodzian** [comment]
Remove the tar.gz file from the pull request.

File: SPECS/libxml2/libxml2.spec [hunk 2]
```diff
@@ -105,6 +102,9 @@ rm -rf %{buildroot}/*
 %{_libdir}/cmake/libxml2/libxml2-config.cmake
 
 %changelog
+* Thu May 27 2021 Mateusz Malisz <mamalisz@microsoft.com> - 2.9.12-1
+- Update to version 2.9.12 to fix CVE-2021-3517 and CVE-2021-3518
```
Comment (jslobodzian) [inline SPECS/libxml2/libxml2.spec:106]: Also fixes CVE-2021-3537

**Review by jslobodzian — APPROVED** [review]
:shipit:

**Comment by anphel31** [comment]
Please also update cgmanifest.json

File: SPECS/libxml2/libxml2.signatures.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
  "Signatures": {
-  "libxml2-2.9.10.tar.gz": "aafee193ffb8fe0c82d4afef6ef91972cbaf5feea100edc2f262750611b4be1f"
+     "libxml2-2.9.12.tar.gz": "c8d6681e38c56f172892c85ddc0852e1fd4b53b4209e7f4ebf17f7e2eae71d92"
```
Comment (anphel31) [inline SPECS/libxml2/libxml2.signatures.json:3]: remove extra spaces

**Pull request merged**